features/step_definitions/appointment_locator_steps.rb

```diff
@@ -7,12 +8,15 @@
   allow(Locations::Repository).to receive(:all).and_return(locations)
   allow(Geocoder).to receive(:coordinates).with(postcode).and_return(lat_lng)
   allow(Locations::Search).to receive(:nearest_to).with(
-    locations, lat_lng, limit).and_return(locations.take(limit))
+    locations, lat_lng, limit).and_return(search_results.take(limit))
 
   Pages::Locations.new.load(postcode: postcode)
 end
 
 Then(/^I should see the (\d+) appointment locations nearest to that postcode$/) do |number_of_locations|
   expected_locations = number_of_locations.to_i.times.map { |i| "cab-#{i + 1}" }
+  expected_distances = number_of_locations.to_i.times.map { |i| "#{i}km" }
```

Comment:
How about we leave that for a presenter spec, when we implement an actual view – we don't want to be partially testing things in cucumber at this stage. Sticking with whole numbers for now keeps us moving forward with driving out the implementation.

Everyone ok with that?